modules/dexertoBidAdapter.md

```diff
@@ -0,0 +1,38 @@
+# Overview
+
+```
+Module Name: Dexerto Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: niels.claes@dexerto.com
+```
+
+# Description
+
+Dexerto currently supports the BANNER type ads through prebid js
+
+Module that connects to dexerto's demand sources.
+
+# Banner Test Request
+```
+  var adUnits = [
+    {
+      code: 'display-ad',
+      mediaTypes: {
+        banner: {
+            sizes: [[300, 250]], 
+        }
+      }
+      bids: [
+        {
+            bidder: 'dexerto',
+            params: {
+                placement_id: 110003,  // Required parameter
```

Comment:
we are responding 100% of time, can you please check it out and let us know if any issue occurs.

thanks.

Reply:
@pranavsheth sorry, after many refreshes I only see 204 or empty 200 responses. this will need to be updated to params that will return valid bids on all requests for prebid testing.

Reply:
@pranavsheth Sorry, I am not seeing this when testing the placement_id you provided in your markdown file. I am seeing your bidUtils intepretResponse function error when reading your no bid 204 response, please fix this and return a bid in your 200 response:
![Screenshot 2024-09-20 at 11 33 05 AM](https://github.com/user-attachments/assets/53c26c33-6c4a-4386-9d84-14198cf7958e)
![Screenshot 2024-09-20 at 11 33 22 AM](https://github.com/user-attachments/assets/3ca961bf-c5a7-4b71-9907-c5e1ccc250a1)

Reply:
HI @ncolletti 
can u send me your request which you are using for testing?

Reply:
Still seeing the same issue. I am adding the bidder params you supplied in your md file to the hello_world.html page in the project's integration examples. there is documentation in the repo to explain how you can run these on your end.

Reply:
request to https://rtb.dexerto.media/hb/dexerto:

```
[{"id":"3f451507-7dd1-4eb7-8ce3-56413e153439","imp":[{"id":"344053ab61566","bidfloor":0,"banner":{"h":250,"w":300}}],"placementId":110003,"site":{"page":"http://localhost:9999/integrationExamples/gpt/hello_world.html?pbjs_debug=true","name":"localhost:9999"},"user":{"id":"","buyeruid":"","keywords":"","customdata":""}}]
```

Reply:
Hi @ncolletti 
can you please confirm that you are able to see ads?
thanks

Reply:
Hello @ncolletti 
we made this changes, can you please review and merge it to the main branch.

Thanks

Reply:
looks good, thank you for making the adjustments